ui/admin/app/templates/scopes/scope/roles/role/scopes.hbs

```diff
@@ -147,6 +147,21 @@
           </B.Tr>
         </:body>
       </Hds::Table>
+    {{else}}
+      <Hds::ApplicationState as |A|>
+        <A.Header @title={{t 'resources.role.scope.messages.title'}} />
+        <A.Body @text={{t 'resources.role.scope.messages.description'}} />
+        {{#if (can 'create model' this.scope collection='roles')}}
```

Comment:
I'm thinking that if the link is going to direct users to the manage-scopes route then maybe the permission we should check is can "set-grant-scopes" on a role?

Reply:
Absolutely. I forgot to update the exact ability. Thanks, I will update now.

Reply:
Fixed now.